.github/workflows/build-timestamped-master.yml

```
@@ -0,0 +1,19 @@
+name: Build
+
+on:
+    push:
+        branches:
+            - master
```

This should be `main`.
```suggestion
            - main
```